doc/source/contributing.rst

```diff
@@ -611,6 +602,21 @@ Or with one of the following constructs::
     pytest pandas/tests/[test-module].py::[TestClass]
     pytest pandas/tests/[test-module].py::[TestClass]::[test_method]
 
+Using `pytest-xdist <https://pypi.python.org/pypi/pytest-xdist>`_, one can 
+speed up local testing on multicore machines. Two scripts are provided to
+assist with this.  These scripts distribute testing across 4 threads.
+
+On Unix variants, one can type::
+
+    test_fast.sh
+    
+On Windows, one can type::
+
+    test_fast.bat
+    
+This can significantly reduce the time it takes to locally run tests before
```

Comment:
@jreback  OK, just pushed that commit.  You probably can kill some of the travis/circleCI/appveyor stuff since there is no code to test here.

Maybe one day pytest-xdist will be available via a standard conda install!